db/migrate/20170822145654_add_user_id_to_posts.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserIdToPosts < ActiveRecord::Migration[5.1]
+  def change
+    add_column :posts, :user_id, :integer
```

Comment:
Question1. What does an index do?
Question2. What does a reference do?
Question3. Wha does an integer column not do?